app/routes/campaignRoute.js

```diff
@@ -0,0 +1,32 @@
+import express from 'express';
+import CampaignController from '../controllers/campaignController';
+
+const router = express.Router();
+const campaignController = new CampaignController();
+
+
+function initRoutes() {
+  router.get('/clients', (req, res) => {  
```

Comment:
Do you really require to add an additional function.
You can do it like this,
router.get(`/clients`, CampaignController.getClients);